tests/phpunit/MigrateTest.php

```diff
@@ -55,7 +55,13 @@ public function testMigrateSuccess(
             [
                 Config::PROJECT_RESTORE_COMPONENT,
                 [
-                    'parameters' => array_merge($expectedCredentialsData, ['useDefaultBackend' => true]),
+                    'parameters' => array_merge(
+                        $expectedCredentialsData,
+                        [
+                            'useDefaultBackend' => true,
+                            'restoreConfigs' => true,
+                        ]
```

Comment:
Ještě prosím přidat mock test na to, že se tam pošle false když je migrateSecrets=true

Reply:
jo